src/site/_variables.scss

```diff
@@ -43,3 +43,7 @@ $error-outline: 2px $error solid;
 $border-light: 1px $background-color solid;
 $border-error: 1px $error solid;
 $border-dark: 1px $light-gray solid;
+
+$field-margin: .25rem 0;
```

Comment:
Don’t create new variables. please use the existing color code or space variables.